openshift/4.0/templates/migrations/migration-job.yaml

```diff
@@ -29,7 +29,41 @@ parameters:
       [Development, Staging, Production].
     required: true
     value: Production
+  - name: DATABASE_ROLE_NAME
```

Comment:
Is this used by Jenkins to run the database migrations when going to prod?
Also I noticed references to the old project namespace in this file.

Reply:
yes, this creates a separate container to run ef just for migrations.